src/Raven.Server/Documents/Patch/PatchDocumentCommand.cs

```diff
@@ -182,7 +182,12 @@ protected PatchResult ExecuteOnDocument(DocumentsOperationContext context, strin
 
                             result.Status = PatchStatus.Created;
                         }
-                        else if (DocumentCompare.IsEqualTo(originalDoc, modifiedDocument, tryMergeMetadataConflicts: true) != DocumentCompareResult.Equal)
+                        else if (DocumentCompare.IsEqualTo(originalDoc, modifiedDocument, 
+                                    new DocumentCompare.DocumentCompareOptions
```

Comment:
Let's make this into a constant, like the `Default` field there

Reply:
done